tests/jballerina-unit-test/src/test/resources/test-src/types/constant/simple-literal-constant-negative.bal

```diff
@@ -321,3 +321,19 @@ Ints2 ints2 = 2;
 
 float:NaN floatNan = 1.0;
 float:Infinity floatInf = 1.0;
+
+// Test arithmatic errors
+const NUM1 = -1;
```

Comment:
Shall we add more tests with different combinations of constants, when both negative, both positive, one negative and one positive etc...?

Reply:
Added 526aee55dfdb85ca0d7e62171ca467a9584f6a48